src/main/java/uk/gov/hmcts/reform/blobrouter/data/eventsv2/NewEnvelopeEvent.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.blobrouter.data.eventsv2;
+
+import uk.gov.hmcts.reform.blobrouter.data.events.EventType;
+
+import java.util.UUID;
+
+public class NewEnvelopeEvent {
+
+    public final UUID envelopeId;
+    public final EventType type;
```

Comment:
Where is the `EventType`?

Reply:
Event was renamed to this https://github.com/hmcts/blob-router-service/pull/337

Reply:
my bad, only now pulled latest master - i was checking for changes locally